plugins/govppmux/vppcalls/vppcalls_api.go

```diff
@@ -15,6 +15,14 @@ type VersionInfo struct {
 	BuildDirectory string
 }
 
+// Release returns version in shortened format YY.MM that describes release.
+func (v VersionInfo) Release() string {
+	if len(v.Version) < 5 {
```

Comment:
What the number represents? Wouldn't it be better as a constant?

Reply:
Just as the comment is tryig to explain, it trims the VPP version from `v19.08-rc0~680-g456d2f9c2~b3246` to `19.08`.

Reply:
Agree in this case